### PR TITLE
Zgłoszenie zadań 1.2-1.5

### PR DESCRIPTION
**Pull request by adacienciala**

### CODE REVIEW

File: src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
+package edu.kis.vh.nursery;
+
+public class DefaultCountingOutRhymer {
+    public static final int MAX_SIZE = 12;
+    public static final int FULL = MAX_SIZE-1;
+    public static final int INITIAL = -1;
+    public static final int DEFAULT = -1;
+
+    private int[] numbers = new int[12];
```
Comment (kowallus) [inline src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java]: Another constant?

File: src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+    private int total = -1;
```
Comment (kowallus) [inline src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java]: Constant again?

File: src/main/java/edu/kis/vh/nursery/FIFORhymer.java [hunk 3]
```diff
@@ -1,21 +1,20 @@
...
-		
-		int ret = temp.countOut();
-		
-		while (!temp.callCheck())
-			
-		countIn(temp.countOut());
-		
-		return ret;
-	}
+    private DefaultCountingOutRhymer temp = new DefaultCountingOutRhymer();
```
Comment (kowallus) [inline src/main/java/edu/kis/vh/nursery/FIFORhymer.java]: should be final

File: src/main/java/edu/kis/vh/nursery/list/IntLinkedList.java [hunk 4]
```diff
@@ -2,39 +2,73 @@
...
-	public int pop() {
-		if (isEmpty())
-			return -1;
-		int ret = last.value;
-		last = last.prev;
-		return ret;
-	}
+    private class Node {
+
+        private int value;
```
Comment (kowallus) [inline src/main/java/edu/kis/vh/nursery/list/IntLinkedList.java]: Immutable - should be final (setter unnecessary)

**Review by kowallus — COMMENTED** [review]
5.5 pts

**Comment by adacienciala** [comment]
Made necessary changes

**Pull request closed without merging**